### DISCUSSION TITLE
Popover scroll issues

### DISCUSSION THREAD

**Discussion by vankhoawin** [twilio-labs/paste#1668] [Q&A] [answered]
Hi team, running into some weirdness about Popover and scrolling. In the first picture you'll see that the Popover is in the correct placement, but after scrolling down the page a bit, it seem to not be attached to the component it encompasses.

Scrolled to the top

![image](https://user-images.githubusercontent.com/6936736/125676609-58c6fbd6-6fbe-4d15-b121-1b3ff2671369.png)

Scrolled a bit down

![image](https://user-images.githubusercontent.com/6936736/125676627-53c0e867-3c19-43a1-b1cd-22cc582f27ce.png)

**Comment by gloriliale** [comment]
Hi @vnguyen94! That is certainly unexpected, thanks for letting us know. Is there any way we could see the code that the screenshots are from?

**Reply by vankhoawin** [reply]
yup! should be up at https://console.twilio.com/?frameUrl=/console/phone-numbers/search

**Reply by vankhoawin** [reply]
oh for the code, let me link you on Slack

**Comment by andioneto** [comment, marked as answer]
Hi @vnguyen94 👋 
To follow up from our conversations on Slack, the tooltip uses `position: absolute;` which requires a relative parent.

Below you can see that the `div` that contains the `iframe` is styled as `position: relative;`:
> <img width="900" alt="container_position" src="https://user-images.githubusercontent.com/18120906/126385019-a9dd7a79-26e0-44d0-9b11-11acb8911b6f.png">

However children of the `iframe` do not inherit styles from the container, so its `position` property is the default value (as it is not explicitly set), which you can see below:
> <img width="975" alt="body_position" src="https://user-images.githubusercontent.com/18120906/126385476-425a53e4-2a63-41b9-813d-472b5d491e39.png">


The tooltip is then relative to the containing `div`, rather than the contents of the `iframe`.
We suggest that the body be styled as `position: relative;` to fix this unexpected performance.

Please let us know if you have anymore questions, and thanks for reaching out! 🙌
